scripts/build-glean-metadata.py

```diff
@@ -446,17 +479,30 @@ def get_app_variant_description(app):
                 "https://raw.githubusercontent.com/mozilla-services/mozilla-pipeline-schemas/generated-schemas/schemas/"  # noqa
                 + bq_path
             ).json()
+            app_channel = app.app.get("app_channel")
             variant_data = dict(
                 id=app_id,
                 description=get_app_variant_description(app),
                 table=stable_ping_table_name,
-                channel=app.app.get("app_channel", "release"),
+                channel=app_channel if app_channel else "release",
             )
-            looker_url = get_looker_explore_url(
-                looker_namespaces, app_name, ping.identifier, stable_ping_table_name
+            looker_explore = (
+                {
+                    "name": "event_counts",
+                    "url": get_looker_event_count_explore_url(
+                        looker_namespaces, app_name, app_channel
+                    ),
+                }
+                if ping.identifier == "events"
+                else {
+                    "name": ping.identifier,
+                    "url": get_looker_ping_explore_url(
+                        looker_namespaces, app_name, ping.identifier, stable_ping_table_name
+                    ),
+                }
             )
-            if not app_is_deprecated and looker_url:
-                variant_data.update({"looker_url": looker_url})
+            if not app_is_deprecated and looker_explore:
```

Comment:
We could probably skip my suggestion in `PingDetail` above if we do this:
```suggestion
            if not app_is_deprecated and looker_explore.get("url"):
```